events/views.py

```diff
@@ -153,7 +156,8 @@ def events_home(request):
         'type': type,
         'offset': offset + 1,
         'nbPage': nbPages,
-        'range': range(1, nbPages + 2)}
+        'range': range(1, nbPages + 2),
+        'search_term': search_term if search_term else ''}
```

Comment:
Is this is useful ? I believe it defaults to None anyhow, and in the template you only check if it exists => `None` is fine